nicegui/ui_run.py

```diff
@@ -53,7 +53,7 @@ def run(*,
     Most of them only apply after stopping and fully restarting the app and do not apply with auto-reloading.
 
     :param host: start server with this host (defaults to `'127.0.0.1` in native mode, otherwise `'0.0.0.0'`)
-    :param port: use this port (default: `8080`)
+    :param port: use this port (default to `8080` not in native mode. If not specify, NiceGUI searches for an open port in native mode)
```

Comment:
The formulation "default to `8080` not in native mode" is not descriptive enough.

Reply:
I changed it to 
```
:param port: use this port (If not specify, default to `8080` not in native mode. If not specify, NiceGUI searches for an open port in native mode)
```